integration-tests-cucumber/src/test/resources/dlab/project.feature

```diff
@@ -1,18 +1,146 @@
 Feature: Project management in DLab
   Such feature allowed to manage projects inside DLab
 
-  Scenario Outline: Create new project when it does not exist
+  Scenario Outline: Create new project when it does not exist and use shared image is enable
 
     Given There is no project with name "<name>" in DLab
     And There are the following endpoints
       | local |
     And There are the following groups
       | $anyuser |
-    And User generate new publicKey
-    And User try to create new project with name "<name>", endpoints, groups and publicKey
-    When User send create new project request
-    Then User wait maximum <timeout> minutes while project is creating
+    And User generates new publicKey
+    When User tries to create new project with name "<name>", endpoints, groups, publicKey and use shared image "true"
```

Comment:
Should we unify the value for the **shared image** parameter? I mean [there](https://github.com/apache/incubator-dlab/pull/512/files#diff-c52ad2bc7daddbcfdf99b83217be5b32R4) you used _enable_, but here "true".